.github/workflows/build.yml

```diff
@@ -18,18 +18,24 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
+    permissions: write-all
```

Comment:
Quickly googling `ERROR: denied: installation not allowed to Write organization package` and the suggestion is to update these permissions to `write-all` as you already have done - so not sure there, but otherwise everything LGTM

Reply:
It's still not working, so I need to work more on this!